util/plugins/eu.esdihumboldt.util.test/META-INF/MANIFEST.MF

```diff
@@ -7,3 +7,5 @@ Fragment-Host: eu.esdihumboldt.util;bundle-version="2.5.0"
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Import-Package: com.vividsolutions.jts.geom;version="1.13.0",
  org.junit;version="4.8.1"
+Require-Bundle: com.google.guava,
```

Comment:
For the test bundle it doesn't matter much but usually I would prefer that you use package imports rather than require bundle, especially in cases where only a small part of a library is actually used.